javascript/kadai/1-2/1-2.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="ja">
+  <head>
+    <meta charset="UTF-8" />
+    <title>①-2</title>
+  </head>
+  <body>
+    <script>
+      let count = 0;
+      while (count === 0) {
```

Comment:
無限ループは必ず条件がtrueになれば良いので、条件そのものをtrueにしてしまうと良いです！
```suggestion
      while (true) {
```
そうすると、countという定義も削除できますね。